test/index.spec.js

```diff
@@ -5,42 +5,67 @@ const preset = require('..');
 
 expect.addSnapshotSerializer(serializerPath);
 
-it('should return a default config that target browsers and node', () => {
-    expect(preset()).toMatchSnapshot();
-});
+describe('targets', () => {
+    it('should return a default config that target browsers and node', () => {
+        expect(preset()).toMatchSnapshot();
+    });
 
-it('should return a config for node if options.targets is set to node', () => {
-    expect(preset(null, { targets: ['node'] })).toMatchSnapshot();
-});
+    it('should return a config for node if options.targets is set to node', () => {
+        expect(preset(null, { targets: ['node'] })).toMatchSnapshot();
+    });
 
-it('should return a config for node if options.targets is set to browser', () => {
-    expect(preset(null, { targets: ['browsers'] })).toMatchSnapshot();
-});
+    it('should return a config for browsers if options.targets is set to browser', () => {
+        expect(preset(null, { targets: ['browsers'] })).toMatchSnapshot();
+    });
 
-it('should allow overriding options.targets with an object', () => {
-    expect(preset(null, { targets: { node: 'current' } })).toMatchSnapshot();
+    it('should allow overriding options.targets with an object', () => {
+        expect(preset(null, { targets: { node: 'current' } })).toMatchSnapshot();
+    });
 });
 
-it('should disable modules if options.modules is disabled', () => {
-    expect(preset(null, { modules: false })).toMatchSnapshot();
-});
+describe('modules', () => {
+    it('should set modules to false if options.modules is disabled', () => {
+        expect(preset(null, { modules: false })).toMatchSnapshot();
+    });
 
-it('should set modules to false if BABEL_ENV is es', () => {
-    process.env.BABEL_ENV = 'es';
+    it('should set modules to false by default if BABEL_ENV is es', () => {
+        process.env.BABEL_ENV = 'es';
 
-    try {
-        expect(preset(null)).toMatchSnapshot();
-    } finally {
-        delete process.env.BABEL_ENV;
-    }
+        try {
+            expect(preset(null)).toMatchSnapshot();
+        } finally {
+            delete process.env.BABEL_ENV;
+        }
+    });
 });
 
 describe('react', () => {
     it('should enable react if options.react is enabled and enable development goodies', () => {
         expect(preset(null, { react: true })).toMatchSnapshot();
     });
 
-    it('should enable react optimizations if env is production', () => {
+    it('should enable react optimizations if options.env is production', () => {
         expect(preset(null, { env: 'production', react: true })).toMatchSnapshot();
     });
 });
+
+describe('namedDefaultExport & babel-plugin-add-module-exports', () => {
+    it('should not enable add-module-exports if options.modules is not commonjs', () => {
+        expect(preset(null, { modules: false })).toMatchSnapshot();
+    });
+
+    it('should enable add-module-exports if options.modules is commonjs', () => {
```

Comment:
"should enable add-module-exports **by default**"

Reply:
I've separated the tests a bit further.